ggshield/core/text_utils.py

```diff
@@ -136,15 +136,28 @@ def file_info(
     filename: str,
     incident_count: int,
     incident_status: IncidentStatus = IncidentStatus.DETECTED,
+    number_of_hidden_secrets: int = 0,
```

Comment:
naming: I would name this `hidden_secret_count`, it's shorter and consistent with `incident_count`.